erpnext_com/www/docs/user/manual/en/setting-up/email/document-follow.md

```diff
@@ -0,0 +1,36 @@
+<!-- add-breadcrumbs -->
+# Document Follow
+
+> Introduced in v12
+
+**Document Follow is feature that allows users to track changes in documents via email notifications.**
+
+## 1. How to Activate Document Follow
+1. Go to: **Settings > My Settings**.
+1. In the Document Follow section, tick the 'Send Notifications for documents followed by me' checkbox.
+1. Set the frequency at which you want to receive emails.
+1. Now, you can receive emails for the documents you follow.
+
+## 1.1 How to Follow documents
+1. Go to any document, you'll see an option to follow the document.
+1. Click on 'Follow'.
+
+<img class="screenshot" alt="Defining Criteria" src="{{docs_base_url}}/assets/img/setup/email/document-follow-how-to-follow.png">
```

Comment:
the red box color is darker than usual, use the default red color

Reply:
done